creator-node/nginx_conf/nginx.conf

```diff
@@ -0,0 +1,44 @@
+worker_processes 1;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    lua_package_path "/usr/local/openresty/conf/?.lua;;";
+
+    proxy_cache_path /usr/local/openresty/cache levels=1:2 keys_zone=cidcache:1000m
+					max_size=10g inactive=10m use_temp_path=off;
+
+    server {
+        listen 4000;
```

Comment:
openresty will run as expected on the first node. will update this to take in env var later

Reply:
looks like you can't dynamically load in the load. https://stackoverflow.com/questions/45517149/environment-variable-to-set-server-port-in-lua

with that said, we should only enable openresty on local cn1 ever. can move the openrestyEnabled env var into cn1 shell script only

Reply:
I think this is still reasonable in terms of local testing

Reply:
> we should only enable openresty on local cn1 ever.
for my own edification - why is this the case?

Reply:
so if openresty is enabled on cn1, the cache layer will be enabled on port 4000 and the app will be enabled on port 3000. this was relatively easy to set up.

 if we enable the cache layer on cn2-10, we just have to change the external-internal mappings of the port (which is definitely doable). this requires changes to the docker-compose.yml, and changes to the `shellEnv#.sh` files. I just figure for simplicity's sake, we should just enable openresty on 1 node (cn1) for local testing purposes!